CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ This gem is moving onto its own [Semantic Versioning](https://semver.org/) schem
 
 Prior to v1.0.0 this gem was versioned based on the `MAJOR`.`MINOR` version of RuboCop. The first release of the ezcater_rubocop gem was `v0.49.0`.
 
+## 5.3.0
+- Add `FeatureFlagNameValid` cop to validate correct feature flag name format, adopted from the cop in `ez-rails`: https://github.com/ezcater/ez-rails/blob/main/cops/rubocop/cops/feature_flags_flag_name.rb.
+
```

Comment:
I went with a minor version bump for now but I could also see an argument for a major version bump since this could require other squads to disable or actively add `TODO`'s through-out their codebases depending on how many feature flags they use.

Reply:
minor bumps are for new features so that tracks, and I believe most of our folks expect some rubocop slicing-and-dicing on upgrades.

Reply:
Ah I did just notice that it says this in the README:
```When adding a new cop, please enable the cop and release a new major version. This allows us to constantly roll out improvements without clients having their suite break unknowingly. 
```

So in the spirit of staying consistent, I'll change this to a major bump instead if that works with everyone.